conans/client/settings_preprocessor.py

```diff
@@ -31,7 +42,7 @@ def _check_cppstd(settings):
 
     # Check that we have a flag available for that value of the C++ Standard
     def check_flag_available(values_range, value, setting_id):
-        available = [v for v in values_range if cppstd_flag(compiler, compiler_version, v)]
+        available = [v for v in values_range if _make_cppstd_flag(compiler, compiler_version, v)]
```

Comment:
Keep using the old fachade of the `cppstd_flag` function, we'll think what to do here when we remove it

Reply:
okay, I've kept old implementation for a while